src/components/molecules/Message/Message.jsx

```diff
@@ -11,16 +11,19 @@ export const MessageComponent = ({ children, date, isYours, profileID }) => {
     const redirect = useNavigate();
     const userDetails = useUserDetails(profileID);
 
-    return (
-        userDetails.isSuccess && (
+    if (userDetails.data) {
+        const { profile_picture_url, displayName } =
+            userDetails.data.data() || {};
+
+        return (
             <div className={`message__field ${isYours ? 'ownMessage' : ''}`}>
                 <ImageComponent
-                    src={userDetails.data.data().profile_picture_url}
+                    src={profile_picture_url || ''}
```

Comment:
Cyprian told me that if user have no profile picture we should display default picture

Reply:
whats the path to the default profile picture?